README.md

```diff
@@ -22,13 +22,13 @@ e.g.
 Instead of typing
 
 ```
-./typo3/cli_dispatch.phpsh extbase extension:install realurl
+typo3/cli_dispatch.phpsh extbase extension:install realurl
 ```
 
 just type:
 
 ```
-./typo3cms extension:activate realurl
+vendor/bin/typo3cms extension:activate realurl
```

Comment:
Since current suggestion is just wrong, we need to hint at the Composer `bin-dir`. I'd expect people to be smart enough to adopt the path if customized.

And yes, this should be changed everywhere actually. Tell me what you think and I'll do the changes.